src/components/DeviceCheckAddress.js

```diff
@@ -22,11 +15,6 @@ class CheckAddress extends PureComponent<Props, State> {
     isVerified: null,
   }
 
-  componentDidMount() {
```

Comment:
mmh doesn't this change remove the purpose of this component tho?
I would suggest we just remove the component & replace all the usages by directly doing what the `render()` do here?

Reply:
It is used in 1 place and I think you are right, it doesn't do much at the moment.

Reply:
nice catch lol